run/events-pubsub/app.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START run_events_pubsub_server_setup]
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
Bodyparser shouldn't necessary anymore, use the built in functionality.

Reply:
The note being that we recently discovered `app.use(express.json())` should be sufficient in lieu of the body-parse dependency. An issue to adjust and test this for existing samples should be filed.

Reply:
Oh, thanks.